### PR TITLE
Support Tizen 2.1 SDK.

### PR DESCRIPTION
**Pull request by pyrasis**
Support Tizen 2.1 SDK.

### CODE REVIEW

File: cocos2dx/platform/third_party/tizen/rootstraps/tizen-emulator-2.1.native/usr/include/curl/curl.h [hunk 1]
```diff
@@ -0,0 +1,2125 @@
+#ifndef __CURL_CURL_H
```
Comment (dumganhar) [inline cocos2dx/platform/third_party/tizen/rootstraps/tizen-emulator-2.1.native/usr/include/curl/curl.h:1]: Why to copy so many tizen relative files( seems header files in tizen system) to `third_party`?
Are these files not exist in tizen sdk?

**Comment by pyrasis** [comment]
Yes. These header and library files are not exist in Tizen SDK version 2.0 and 2.1.

libraries

**Comment by dumganhar** [comment]
So you build them by yourself?

**Comment by pyrasis** [comment]
The library files were copied from Tizen Emulator image.
and the header files were generated from Tizen GBS(Git Build System) package.

**Comment by dumganhar** [comment]
Thanks.

**Pull request merged**